### PR TITLE
Fix types in SwapAssetSelector

### PR DESCRIPTION
**Pull request by tiltom**

### CODE REVIEW

File: src/app/components/Form/AssetSelect/index.tsx [hunk 1]
```diff
@@ -26,10 +26,8 @@ export function AssetSelect(props: Props) {
   return (
     <Select
       value={props.value as any}
```
Comment (ShynRou) [inline src/app/components/Form/AssetSelect/index.tsx:28]: Select should accept a nullish value, no casting should be required.

**Review by ShynRou — CHANGES_REQUESTED** [review]

File: src/app/components/Form/AssetSelect/index.tsx (hunk 1 above)
Comment (tiltom) [reply src/app/components/Form/AssetSelect/index.tsx:28]: I wanted to change it but unfortunately, we don't have any current usage of this. However, I can change it and we can adjust the usage later.

Comment (ShynRou) [reply src/app/components/Form/AssetSelect/index.tsx:28]: Okey.

**Review by ShynRou — APPROVED** [review]

File: src/app/components/Form/AssetSelect/index.tsx (hunk 1 above)
Comment (tiltom) [reply src/app/components/Form/AssetSelect/index.tsx]: Changed + `Select` was adjusted as well.

**Review by ShynRou — APPROVED** [review]

**Review by creed-victor — APPROVED** [review]

**Pull request merged**